backend/README.md

```diff
@@ -311,90 +305,54 @@ Run with (supervisord.conf should be provided, see [supervisord.conf default_pat
-    python3 ./setup_dev_env.py
-
-Before running some functional test related to email, you need a local working *MailHog*
-see here: https://github.com/mailhog/MailHog
-
-You can run it this way with Docker:
-
-    docker pull mailhog/mailhog
-    docker run -d -p 1025:1025 -p 8025:8025 mailhog/mailhog
-
-You need also a test ldap server on port 3890 for ldap related test.
-See here: https://github.com/rroemhild/docker-test-openldap
-
-You can run it this way with Docker:
-
-    docker pull rroemhild/test-openldap
-    docker run -d -p 3890:389 rroemhild/test-openldap
-
-You need also a elasticsearch server on port 9200 for elasticsearch related test
-You can run it this way with Docker Compose:
+Install the required versions:
 
-    # from backend folder
-    docker-compose up elasticsearch
+    pip install -r requirements-devtool.txt
 
-You also need a minio server on port 9000 for tests related to s3.
-You can run it this way with Docker Compose:
+Then run flake8:
 
-    # from backend folder
-    docker-compose up minio
+    flake8
 
-please note that syntax is similar to minio command line, "minio server /data" will start the minio server and use
-/data dir (in our case, it's the /data of the docker and his content will be deleted after stopped the docker as we do not
-create a docker volume), see minio documentation for more info: https://docs.min.io/docs/minio-docker-quickstart-guide.html .
+## Run Tests
 
-Create default test_storage_dir folder tree to make test work properly out of the box:
+First setup the needed tools and directories (only needed once):
 
+    # from backend directory
+    python3 ./setup_dev_env.py
     ./create_test_storage_dir.sh
+    # several external services (mail/databases/…) are started through a docker compose file
+    sudo apt install docker.io docker-compose
+    # add the current user to docker group, you'll need to use a new login shell for this change to be taken into account
+    sudo usermod -a -G docker $USER
 
-Run your project's tests:
+Running tests can be done with:
 
+    docker-compose up -d
     pytest
+    docker-compose down
 
-### Linting and Other Checks ###
-
-Install the required versions:
-
-    pip install -r requirements-devtool.txt
-
-Run mypy checks:
-
-    mypy --ignore-missing-imports --disallow-untyped-defs tracim_backend
-
-Code formatting using black:
 
-    black -l 100 tracim_backend
+By default the tests will be executed with the `sqlite` database, this can be changed using the `--database` argument of pytest:
 
-Sorting of import:
+    pytest --database=postgresql
 
-    isort tracim_backend/**/*.py
+Possible databases are `sqlite`, `postgresql`, `mysql` and `mariadb`.
+It is possible to specify several databases or even `all`:
 
-Flake8 check(unused import, variable and many other checks):
+    # Needs the pytest-forked plugin (pip install pytest-forked)
+    # as some test fixtures do modify static variables.
+    pytest --forked --database=sqlite --database=postgresql
```

Comment:
Does running on multiple database work in parallel ? if yes, are we sure there is no concurrency issue
database is one thing but : 
- depot storage should be independant.
- we do not expect multiple process to deal with mailhog (we do clean the mailhog database for example)
- same for openldap (but i don't think this one can cause issue)
- caldav storage should also be independant.

Reply:
No it doesn't run in parallel with `--forked`, the option does only start each test in a new process, but stiil runs them sequentially.